manifests/init.pp

```diff
@@ -115,14 +122,24 @@
 
   if $nfs_service == 'USE_DEFAULTS' {
     $nfs_service_real = $default_nfs_service
+    $nfs_service_required_svcs_real = $default_nfs_service_required_svcs
   } else {
     $nfs_service_real = $nfs_service
+    $nfs_service_required_svcs_real = undef
   }
 
   package { $nfs_package_real:
     ensure => present,
   }
 
+  if $nfs_service_required_svcs_real and $nfs_service_real {
```

Comment:
Better to be explicit

`if $nfs_service_required_svcs_real != undef and $nfs_service_real != undef`